Polyfills/Canvas/Source/nanovg/nanovg_babylon.cpp

```diff
@@ -637,6 +649,7 @@ namespace
                     );
                 gl->encoder->setVertexBuffer(0, &gl->tvb);
                 gl->encoder->setTexture(0, gl->s_tex, gl->th);
+                gl->encoder->setTexture(1, gl->s_tex2, gl->th2);
```

Comment:
Had to add these in since the metal backend would complain if nothing is set to `s_tex2`

Reply:
A texture was still bound in tex2 (text + gradient) but the shader doesn't use it. There was a warning with D3D11 that's now fixed with this change.